src/docs/workflow.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: "Workflow"
+---
+
+You may also be interested in <Link to="/sentry-vs-getsentry/">Sentry vs Getsentry</Link>.
+
+
+## Testing Sentry commits in Getsentry
```

Comment:
I feel like this might as well go into the CI section, otherwise this page feels kinda empty

Reply:
Ah, that's an equally suitable place. I wanted to expand the workflow section in the near future though (how to bump python dependencies).

Reply:
I'll merge this first, then decide later.